asyncAPICalls.js

```diff
@@ -0,0 +1,14 @@
+async function fetchAPICalls() {
+    let response = await fetch("https://reqres.in/api/users");
+    let result = await response.json();
+    const dataArray = result.data;
+    
+    dataArray.forEach((user, index) => {
+        if (index < 3) {
+            console.log(`${user.first_name} ${user.last_name}`);
+        }
+        
+    });
```

Comment:
Very clean and elegant fetching, and data iteration. I especially like how you wrote the callback function in the `forEach()` call. But, probably by accident, you forgot to actually use this data to add it to your HTML, as the exercise asked. 😓 But based on your Ex2 solution, you must be able to do it. You just forgot to. But the rest of the solution is great! 🎖️

Reply:
> Very clean and elegant fetching, and data iteration. I especially like how you wrote the callback function in the `forEach()` call. But, probably by accident, you forgot to actually use this data to add it to your HTML, as the exercise asked. 😓 But based on your Ex2 solution, you must be able to do it. You just forgot to. But the rest of the solution is great! 🎖️


Hello:)
Thanks for your time and comments. I will go through the comments and improve the code.
Sorry, I have a question regarding javascript3 week3 homework, do I have to create a separate html for users and screenshots? Also, if I have separate html and javascript files how do i access the same variable in both js files, I have tried some ways, but it doesn't work.